src/styles.js

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet}  from "react-native";
+import { Colors } from 'react-native/Libraries/NewAppScreen';
```

Comment:
Here you use single quotes and line above double quotes, it should be consistent